src/boot/efi/pe.c

```diff
@@ -162,6 +164,18 @@ static bool pe_section_name_equal(const char *a, const char *b) {
         return true;
 }
 
+static size_t pe_count_sections(
+                const PeSectionHeader section_table[],
+                size_t n_section_table,
+                const char *const section_name) {
+        size_t n = 0;
+        FOREACH_ARRAY(i, section_table, n_section_table) {
+                if (pe_section_name_equal((const char*) i->Name, section_name))
+                        n++;
+        }
+        return n;
```

Comment:
you don't actually care about the number, do you? all that matters is to know if this is a multi-dtb image or not. hence maybe make this return a boolean and call it `pe_is_multi_dtb_uki()`